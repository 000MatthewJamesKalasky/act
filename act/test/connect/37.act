
defproc test(bool x[2][4])
{

}


bool w[4][4];

test t(w[0..1]);
