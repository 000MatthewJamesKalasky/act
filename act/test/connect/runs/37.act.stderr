ERROR: File `37.act', line 10, col 13
	Typechecking failed on expression!
	Identifier `w' has an invalid number of array dimensions (expected 2 v/s 1)
